release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1309,6 +1313,63 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
+
+* Previously, {product-title} release images were not visible in certain registries, such as {rhq-short}. This prevented users from installing {product-title} due to the missing release images. With this update, release images are always tagged to ensure they appear in registries like {rhq-short}, enabling proper installation. (link:https://issues.redhat.com/browse/OCPBUGS-36410[*OCPBUGS-36410*])
+
+* Previously, when using the technology preview oc-mirror plug-in v2 feature in Mirror-to-Disk mode, catalog images and contents were stored in `subfolders` under `working-dir`, based on the image digest. During the Disk-to-Mirror process in fully disconnected environments, oc-mirror tried to resolve the catalog image tag through the source registry, which was unavailable, leading to such errors:
+[source.terminal]
++
+----
+[ERROR] : [OperatorImageCollector] pinging container registry registry.redhat.io: Get "http://registry.redhat.io/v2/": dial tcp 23.217.255.152:80: i/o timeout
+----
++
+With this update, `oc-mirror` checks the local cache during the Disk-to-Mirror process to determine the digest, avoiding the need to query the registry. (link:https://issues.redhat.com/browse/OCPBUGS-36214[*OCPBUGS-36214*])
+
+* Previously, the technology preview oc-mirror plug-in v2 feature failed to mirror operator catalogs from disk-to-mirror in fully disconnected environments. This issue also affected catalogs that specified a `targetCatalog` in the `ImageSetConfiguration` file. With this update, oc-mirror v2 can now successfully mirror catalogs in fully disconnected environments, and the `targetCatalog` functionality works as expected. (link:https://issues.redhat.com/browse/OCPBUGS-34521[*OCPBUGS-34521*])
+
+* Previously, the `oc adm must-gather` command took a long time to gather CPU related performance data in large clusters. With this release, the data is gathered in parallel instead of sequentially, which shortens the data collection time. (link:https://issues.redhat.com/browse/OCPBUGS-34360[*OCPBUGS-34360*])
+
+* Previously, in the technology preview oc-mirror plug-in v2 feature, errors encountered on registries, such as timeouts, expired authentication tokens, HTTP 500 errors, and so on were not retried automatically. With this update, retries for these errors have been implemented. With this update, retries for these errors are implemented, and users can configure retry behavior with the following flags:
++
+* `--retry-times`: Specifies the number of retry attempts. Default is 2.
+* `--retry-delay`: Sets the delay between retries. Default is 1 second.
+* `--image-timeout`: Defines the timeout period for mirroring an image. Default is 10 minutes.
+* `--max-parallel-downloads`: Controls the maximum number of layers to pull simultaneously during a single copy operation. Default is 6.
```

Comment:
Not sure this is the best place for this info; should be in the docs, right? Maybe a link here if it is?

Reply:
It should be, but I'm not seeing it in the flag reference table for oc-mirror plug-in v2. https://docs.openshift.com/container-platform/4.16/installing/disconnected_install/about-installing-oc-mirror-v2.html#oc-mirror-command-reference-v2_about-installing-oc-mirror-v2

Reply:
I created a jira for them to be added.

Reply:
Can you link me the jira? I can take a look at the docs after GA